eldritch/eldritch.py

```diff
@@ -358,7 +358,11 @@ def clear_usables(self):
   def get_interrupts(self, event):
     interrupts = []
     if isinstance(event, events.MoveOne):
-      nearby_monsters = [mon for mon in self.monsters if mon.place == event.character.place]
+      nearby_monsters = [
+          mon
+          for mon in self.monsters
+          if mon.place == event.character.place and mon not in event.character.avoid_monsters
```

Comment:
I notice you are doing the filtering for character.avoid_monsters both here before you create the EvadeOrFightAll event, and inside the `__init__` of EvadeOrFightAll. You probably only need one?

Reply:
Same comment still applies. I think EvadeOrFightAll probably works better, since you haven't made the same modifications to get_triggers.